src/lib/stateMappings/simple.ts

```diff
@@ -1,15 +1,18 @@
-import { UiState, RouteState, StateMapping } from '../../types';
+import { UiState, StateMapping } from '../../types';
 
-class SimpleUIStateMapping implements StateMapping {
-  public stateToRoute(uiState: UiState): RouteState {
-    return uiState;
-  }
+// technically a URL could contain any key, since users provide it,
+// which is why the input to this function is UiState, not something
+// which excludes "configure" as this function does.
+export default function simpleStateMapping(): StateMapping<UiState> {
+  return {
+    stateToRoute(uiState) {
+      const { configure, ...trackedRouteState } = uiState;
```

Comment:
We have to keep in mind that we have to update the logic once we have the new format of the `uiState`.

Reply:
the type UiState will change (https://github.com/algolia/instantsearch.js/pull/4071, but different) so this should fail because of that :)